### DISCUSSION TITLE
Can i make this! by using FPDF2

### DISCUSSION THREAD

**Discussion by ritiksoni00** [py-pdf/fpdf2#354] [Q&A] [answered]
may table is gonna be dynamic.

so I'm quite unsure if i can achieve what i want...

1. can I use CSS property like flex stuff??
2. will it affect my CSS style if i run my program on diff browsers?

![Screenshot from 2022-03-08 14-26-52](https://user-images.githubusercontent.com/47344024/157202948-cc7a15e1-2313-446d-a7a2-e44d9f275a4c.png)

**Comment by Lucas-C** [comment, marked as answer]
Hi @ritiksoni00

> 1. can I use CSS property like flex stuff??

No, `fpdf2` does not support CSS flex, only basic HTML features: https://pyfpdf.github.io/fpdf2/HTML.html#supported-html-features

You may want to check WeasyPrint if you need a broader support for HTML features: https://weasyprint.org

> 2. will it affect my CSS style if i run my program on diff browsers?

`fpdf2` is a Python library, it generates PDFs and does not "run" in a browser.
The PDFs generated by `fpdf2` should be rendered very similarly by different browsers,
but that only depends on their underlying PDF rendering engine (generally: [PDF.js](https://mozilla.github.io/pdf.js/))

**Reply by ritiksoni00** [reply]
thanks for the reply, Lucas!
